src/pudl/extract/eia176.py

```diff
@@ -0,0 +1,26 @@
+"""Extract EIA Form 176 data from CSVs.
+
+The EIA Form 176 archive also contains CSVs for EIA Form 191 and EIA Form 757.
+"""
+
+from dagster import asset
+
+from pudl.extract.csv import CsvExtractor, get_table_file_map
+
+DATASET = "eia176"
+
+
+@asset(required_resource_keys={"datastore"})
+def raw_eia176__company(context):
```

Comment:
I wrote a concrete asset like this for now, after trying to get a factory pattern down. Happy to learn more about the Dagster components and write a factory at some point.